### ISSUE TITLE
Documentation/Example needed: Using Pages and App router simultaneously with next-intl

### ISSUE THREAD

**Issue by mohitb35** [amannn/next-intl#602]
### Is your feature request related to a problem? Please describe.

I'm trying to migrate to the app router - currently all pages in my app use the pages router. 

I tried out the examples for app router and pages router individually and they worked. If I combine them however (which is required to do a gradual migration), things break. 

I either run into a multiple redirect issue (when the locale is part of the url), or the locale detection does not seem to work. 

### Describe the solution you'd like

A minimal example you can provide to use app router and pages router simultaneously with next-intl would be useful. 

### Describe alternatives you've considered

I've searched for a possible solution within [the next-intl docs ](https://next-intl-docs.vercel.app/), github issues on the next-intl repo, and stackoverflow, but have not found anything.

**Comment by amannn** [comment]
A user has contributed an example in the RSC branch: https://github.com/amannn/next-intl/tree/feat/next-13-rsc/examples/example-next-13-with-pages. There's a small note on https://next-intl-docs.vercel.app/docs/getting-started/app-router-server-components to this example, but I agree that it should get more visibility.

Can you have a look if this is helpful to you?

I've added a note to https://github.com/amannn/next-intl/pull/149 to give this example more prominence.

**Comment by mohitb35** [comment]
Thanks for the quick response @amannn .

I'll be able to try this out only after 22nd Nov as I have a few urgent tasks to complete. Will get back if it works, or if I need some clarification.

**Comment by amannn** [comment]
Sounds good! I'll close this ticket for the time being then, since we already have an example and the new docs will feature it more prominently. In case something doesn't work as expected, feel free to open a new issue!
